packages/rancher-vsphere-csi/generated-changes/patch/values.yaml.patch

```diff
@@ -3,9 +3,9 @@
 @@ -150,7 +150,7 @@
  # https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#kubernetes-versions-compatible-with-vsphere-container-storage-plugin-1
  versionOverrides:
-   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.1.2/manifests/vanilla/vsphere-csi-driver.yaml
--  - constraint: ">= 1.26 < 1.29"
-+  - constraint: ">= 1.26 <= 1.29"
+   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.2.0/manifests/vanilla/vsphere-csi-driver.yaml
+-  - constraint: ">= 1.27 < 1.30"
++  - constraint: ">= 1.27 <= 1.30"
```

Comment:
Proper 1.30 support is coming via https://github.com/rancher/rancher/issues/45747#issuecomment-2197578204 as per https://suse.slack.com/archives/C0383T4RAFP/p1719606102540399?thread_ts=1719358426.379069&cid=C0383T4RAFP

We should probably wait on that.

Reply:
@dereknola https://github.com/rancher/vsphere-charts/pull/82 has been merged, this PR can be updated now.

Reply:
Updated